viite-UI/src/view/MapView.js

```diff
@@ -139,12 +139,12 @@
     });
 
     $('body').on('keydown', function(evt){
-      if(evt.shiftKey && enableShiftModifier)
+      if (evt.ctrlKey && enableCtrlModifier)
         map.getViewport().style.cursor = "copy";
     });
 
     $('body').on('keyup', function(evt){
-      if(evt.which === 16) // shift key up
+      if (evt.which === 17) // ctrl key up
```

Comment:
I'm wondering if there is keyCode value or something for ctrl key so we wouldn't have to use magic number 17 here.

Reply:
Something like evt.ctrlKey could also work.

Reply:
We should have something like evt.metaKey || evt.ctrlKey to support MAC.